inc/ProgEvent.h

```diff
@@ -114,9 +114,7 @@ namespace PCOE {
 
         std::string timeOfEventUnit; ///< Unit of the ToE
 
-    private:
-        unsigned int nSamples; ///< Number of samples in the occurrence matrix
-        ///@todo(CT): IS THIS NECESSARY?
+        std::vector<UData> eventState;
```

Comment:
Why is it a vector? there is only one event state per event, so this can be a UData